apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -298,8 +299,21 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
                 }
 
                 if (values.credentialId) {
-                  details = { ...details, credentialId: values.credentialId };
+                  details = {
+                    ...details,
+                    credentialId: values.credentialId,
+                  };
                 }
+
+                if (values.teamName) {
+                  details = {
+                    ...details,
+                    teamName: values.teamName,
+                  };
+                }
```

Comment:
Save the credentialId and the team name to the event type location. That way users know who owns the conferencing app that is selected. 

Since team names rarely change, we don't have to query the DB for a team name every time.